server/services/GoService.js

```diff
@@ -5,6 +5,7 @@ import GoTestService from './GoTestService';
 import DBService from './DBService';
 import Logger from '../utils/Logger';
 import CucumberParser from '../utils/CucumberParser';
+import _ from 'lodash'
```

Comment:
Could you add this to `package.json` as well? Now it's in `node_modules` because other dependencies rely on it. Or since you just use it for just one function, could you write in in plain javascript? I like to keep dependencies as few as possible

Reply:
Thanks for the feedback! Turns out a solution without lodash is actually simpler :) Fixed.